test/functional/rpc_blockchain.py

```diff
@@ -307,6 +309,17 @@ def assert_waitforheight(height, timeout=2):
         assert_waitforheight(current_height)
         assert_waitforheight(current_height + 1)
 
+    def _test_getblockbyheight(self):
+        self.log.info("Test getblockbyheight")
+        node = self.nodes[0]
+        node.add_p2p_connection(P2PInterface())
+
+        self.log.info("Getting block with height 20")
+        b20 = node.getblockbyheight(20)
```

Comment:
Something like the following should be enough:
```py
assert_equal(getblock(getblockhash(20)), getblockbyheight)
```
Then you could also test these errors:
 - calling without arguments
 - calling with extra arguments
 - calling with wrong type (string for instance).

Reply:
Thank you, I will edit it

Reply:
You should assert the whole response, not just the hash.